test/timer.cpp

```diff
@@ -76,7 +76,11 @@ BOOST_AUTO_TEST_CASE(timer_start_stop)
 
     BOOST_REQUIRE_EQUAL(tick_count, required_tick_count);
 
+#ifdef WIN32
+    BOOST_REQUIRE_LE(abs(test_duration - required_duration), milliseconds(16*required_tick_count));
+#else
     BOOST_REQUIRE_LE(abs(test_duration - required_duration), milliseconds(10));
+#endif
```

Comment:
Yip, __minimum__ could work, I was thinking of getting statistics and setting the threshold to [~4.5 sigma](http://en.wikipedia.org/wiki/Six_Sigma#Sigma_levels). Just discussing... I imagine it would be for another jira ticket.